packages/@okta/vuepress-site/docs/reference/rl-clientbased/index.md

```diff
@@ -0,0 +1,146 @@
+---
+title: Client-based rate limits
+excerpt: >-
+  Understand what client-based rate limits at Okta and learn how to design for efficient use of resources
+---
+
+# Client-based rate limits
+
+Client-based rate limiting for the `/authorize` endpoint provides granular isolation between requests made to the `/authorize` endpoint using a combination of the Client ID, user's IP address, and Okta device identifier. This framework isolates rogue OAuth clients and bad actors, thereby ensuring valid users and applications don't run into rate limit violations.
```

Comment:
Same comment as for https://github.com/okta/okta-developer-docs/pull/1467/files#diff-b824ac7f01672ca523e3053b27302b62 above